tests/libs/estdlib/test_lists.erl

```diff
@@ -321,4 +323,14 @@ test_mapfoldl() ->
     ?ASSERT_ERROR(lists:mapfoldl(fun(X, A) -> {X * A, A + 1} end, 1, foo), function_clause),
     ok.
 
+test_append() ->
+    ?ASSERT_MATCH(lists:append([]), []),
+    ?ASSERT_MATCH(lists:append([[1, 2, 3, 4]]), [1, 2, 3, 4]),
+    ?ASSERT_MATCH(lists:append([[1, 2], [3, 4]]), [1, 2, 3, 4]),
+    ?ASSERT_MATCH(lists:append([[1, 2], [a, b]]), [1, 2, a, b]),
+    ?ASSERT_MATCH(lists:append([["1", "2"], [a, b]]), ["1", "2", a, b]),
+    ?ASSERT_ERROR(lists:append(1), badarg),
```

Comment:
You had the first one right. It is only the second one that was a badarg.

Reply:
There is no lists:append/1 so it’s a function_clause, the second has 2 bad parameters, so it raises `badarg`